src/mongocxx/test/CMakeLists.txt

```diff
@@ -40,7 +37,7 @@ set(spec_test_common
     spec/util.cpp
 )
 
-set(client_helpers_cpp ${PROJECT_SOURCE_DIR}/lib/mongocxx/v_noabi/mongocxx/test_util/client_helpers.cpp)
+set(client_helpers_cpp client_helpers.cpp)
```

Comment:
Suggest removing the `client_helpers_cpp` variable and replace references with `client_helpers.cpp`.